### PR TITLE
Better error for reused pcollections.

### PR DESCRIPTION
**Pull request by robertwb**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/BEAM/issues/) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

File: sdks/python/apache_beam/transforms/ptransform.py [hunk 1]
```diff
@@ -374,6 +374,12 @@ def __ror__(self, left, label=None):
       p = pipeline.Pipeline(
           'DirectRunner', PipelineOptions(sys.argv))
     else:
+      if any(getattr(p, 'is_ephemeral', False) for p in pipelines):
```
Comment (KesterTong) [inline sdks/python/apache_beam/transforms/ptransform.py]: should this be `True`?

**Comment by aaltay** [comment]
@robertwb any updates on this PR?

**Comment by reuvenlax** [comment]
@robertwb ping

**Comment by aaltay** [comment]
Can we close this PR?

File: sdks/python/apache_beam/transforms/ptransform.py (hunk 1 above)
Comment (robertwb) [reply sdks/python/apache_beam/transforms/ptransform.py]: No, this should be False, which is the correct value if it's not present.

**Pull request closed without merging**